umap/static/umap/js/umap.browser.js

```diff
@@ -129,16 +129,16 @@ L.U.Browser = L.Class.extend({
       makeDirty: false,
       callback: (e) => {
         if (e.helper.field === 'options.inBbox') {
-          if (this.options.inBbox) this.map.on('moveend', appendAll)
-          else this.map.off('moveend', appendAll)
+          if (this.options.inBbox) this.map.on('moveend', rebuildHTML)
+          else this.map.off('moveend', rebuildHTML)
         }
-        appendAll()
-        resetLayers()
+        redrawDataLayers()
```

Comment:
Here are the two switched lines :)

Reply:
Yes, with the renaming, that was tricky to catch!

I thought about making two commits for that, but… I know you'd catch it anyway :p